### PR TITLE
Update versions for 6.0.0 release

### PR DESCRIPTION
**Pull request by markcowl**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by markcowl** [comment]
signing: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/505/

**Comment by markcowl** [comment]
signing for rc3(DDOS): https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/509/

File: src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/Azure.AnalysisServices.psd1 [hunk 1]
```diff
@@ -48,7 +48,7 @@ CLRVersion='4.0'
 ProcessorArchitecture = 'None'  
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '4.6.0'; })
+RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '4.6.1'; })
```
Comment (maddieclayton) [inline src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/Azure.AnalysisServices.psd1]: Missed the bump to 5.0.0 here for Profile - could you bump this in the next commit?

File: ChangeLog.md [hunk 2]
```diff
@@ -37,6 +164,9 @@
 #### AzureRM.EventHub
 * Fixed AlternameName in New-AzureRmEventHubGeoDRConfiguration
 
+#### AzureRM.IotHub
+* Enable tags and Basic Sku to the IotHub
+
```
Comment (maddieclayton) [inline ChangeLog.md]: Not sure how IotHub got put down here - can you move this up to the current release?

**Pull request closed without merging**